### ISSUE TITLE
Copy method summary for abstract methods (c#)

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#54928]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Copy-method-summary-for-abstract-methods/1454922)._

---
When implementing an abstract method, the summary comments of the abstract method should be automatically added above the newly created implementing method.

---
#### Feedback Bot on 6/20/2021, 07:55 PM: 

Thank you for taking the time to provide your suggestion.  We will do some preliminary checks to make sure we can proceed further.  We'll provide an update once the issue has been triaged by the product team.

**Comment by CyrusNajmabadi** [comment]
I don't think we want this.  There is inheritdoc, and the IDE automatically cascades this for the VS experience.  Closing out.  @sharwell  to reopen if he disagrees that this current state is not sufficient and we need to do work ehre.
